manifest.json

```diff
@@ -267,7 +267,7 @@
     },
     {
       "projectID": 429235,
-      "fileID": 3485636,
+      "fileID": 3435176,
```

Comment:
This one seesm to be the fileID for ferritecore 2.0.7. Before this PR we used to have 2.1.0 and the lastest one is 2.1.1. Please take a look at it again to make sure we have the best option here.